test/firebase_io_test.dart

```diff
@@ -11,66 +13,63 @@ Uri getTestUrl(int count, {DateTime timeKey}) => getTestUrlBase(
 //TODO: add tests that validate security by having secured and insecure sections
 void main() {
   int count = 0;
-  group('get and put', () {
+  group('get, put and post', () {
     test('without security', () async {
       var baseUri = getTestUrl(count++);
-
```

Comment:
undo all of this whitespace cleanup – ideally your commit touches minimal other code